cfme/tests/v2v/test_v2v.py

```diff
@@ -0,0 +1,64 @@
+"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
+import fauxfactory
+import pytest
+
+from cfme.utils.log import logger
+
+pytest.mark.ignore_stream('5.8', '5.9')
+
+
+def test_basic_navigation(appliance):
+    """Test takes an appliance and tries to navigate and fill the form for Infra Map and Migration Plan.
+
+    This is a dummy test just to check navigation.
+    """
+    form_data = {
+        'general_view':
+            {
+                'name': 'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                'description': fauxfactory.gen_string("alphanumeric", length=50)
+            },
+        'clusters_view':
+            {
+                'mappings': [
+                    {
+                        'sources': ['Datacenter \ Cluster'],
+                        'target': ['Default \ Default']
+                    }
+                ]
+            },
+        'datastores_view':
+            {
+                'Cluster (Default)': {
+                    'mappings': [
+                        {
+                            'sources': ['NFS_Datastore_1'],
+                            'target': ['hosted_storage']
+                        },
+                        {
+                            'sources': ['h02-Local_Datastore-8GB', 'h01-Local_Datastore-8GB'],
+                            'target': ['env-rhv41-01-nfs-iso']
+                        }
+                    ]
+                }
+            },
+        'networks_view':
+            {
+                'Cluster (Default)': {
+                    'mappings': [
+                        {
+                            'sources': ['VM Network', 'VMkernel'],
+                            'target': ['ovirtmgmt']
+                        },
+                        {
+                            'sources': ['DPortGroup'],
+                            'target': ['Storage - VLAN 33']
+                        }
+                    ]
+                }
+            }
+    }
+
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    add_infrastucture_mapping = infrastructure_mapping_collection.create(form_data)
+    logger.info(add_infrastucture_mapping)
```

Comment:
**Question / Required**

Flash message assertion on the view after create?

Reply:
There is no flash message to assert. I maybe able to add a text widget which can read the text "successfully mapped..." to assert the success?